### PR TITLE
Add display-mode media query

### LINKED ISSUE #6071: Fix false positives for display-mode in media-feature-name-no-unknown
> What is the problem you're trying to solve?

The media-feature-name-no-unknown rule doesn't allow the new [display-mode](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/display-mode) media query feature. 

> What solution would you like to see?

The media query display-mode to be allowed by the media-feature-name-no-unknown rule.

### PR DESCRIPTION
**Pull request by andreypelykh**
This is a new media query added to CSS Media Queries Level 5 to test the display mode of an application.

Closes #6071

### CODE REVIEW

File: lib/reference/keywordSets.js [hunk 1]
```diff
@@ -677,6 +677,7 @@ keywordSets.mediaFeatureNames = uniteSets(keywordSets.deprecatedMediaFeatureName
 	'color-gamut',
 	'color-index',
 	'dynamic-range',
+	'display-mode',
```
Comment (ybiquitous) [inline lib/reference/keywordSets.js:680]: Sort in alphabetical order, please?

```suggestion
	'display-mode',
	'dynamic-range',
```

**Review by ybiquitous — COMMENTED** [review]
@andreypelykh Thank you for opening the PR, and it works expectedly. 👍🏼 

Could you address the small suggestion, please?

File: lib/reference/keywordSets.js (hunk 1 above)
Comment (andreypelykh) [reply lib/reference/keywordSets.js]: @ybiquitous You are welcome! Thanks for the suggestion.

**Review by ybiquitous — APPROVED** [review]
@andreypelykh Thanks, LGTM 👍🏼

**Comment by ybiquitous** [comment]
Changelog entry added:

> Fixed: media-feature-name-no-unknown false positives for display-mode (#6073).

**Pull request merged**